components/BackdropMap.tsx

```diff
@@ -12,25 +12,19 @@ export default function BackdropMap() {
     const [zoom, setZoom] = useState(9)
 
   useEffect(() => {
-    if (map.current) return;
-
     map.current = new mapboxgl.Map({
-      container: mapContainer.current || "",
+      container: mapContainer.current!,
       style: "mapbox://styles/mapbox/streets-v12",
       center: [lng, lat],
       zoom: zoom
```

Comment:
in the case of assigning an object key with a var of the same name, you can just:
```suggestion
      zoom
```